go.mod

```diff
@@ -0,0 +1,24 @@
+module github.com/alphagov/paas-accounts
```

Comment:
We have both `dep` and `go mod` package management files here. Which are we using?